coralproject/talk
# Mocking stream endpoint

## What does this PR do?

Creates a stream endpoint that returns a static json object and adjusts the frontend to expect data of that format.

## How do I test this PR?

1. `npm run build`
2. `npm start`
3. Visit `http://localhost:3000/client/coral-embed-stream/samplearticle.html`
4. The page should render appropriately (though you won’t be able to post comments.)
5. Your server logs should say ‘Stream endpoint has been hit with asset_id assetTest’. (This is temporarily there for verification.)

## Notes

- I went ahead and added an `npm build` script for efficiency, someone with more of a handle on the architecture (i.e. @wyattjoh  or @okbel) should feel free to refactor.
- I went ahead and set up a path for hosting static built files. Same as above, feel free to refactor.
